system/CLI/Commands.php

```diff
@@ -173,20 +175,21 @@ public function discoverCommands()
 				$class = new $className($this->logger, $this);
 
 				// Store it!
-				if ($class->group !== null)
+				if (! is_null($class->group))
 				{
 					$this->commands[$class->name] = [
 						'class'       => $className,
 						'file'        => $file,
 						'group'       => $class->group,
+						'name'	      => $class->name,
```

Comment:
Is there a benefit of adding this line? As I see it, the command's name is already used as the array key and can be accessed later by foreach using `foreach ($commands as $name => $command)`, which is the current way.